wpeverest/user-registration
# Feature - Gutenberg block for login

### All Submissions:





Previously, We have Gutenberg block only for the registration form. In this PR we have added an option for login and my account form.

### How to test the changes in this Pull Request:

1. Goto Page and click on +( Plus ) icon and click on user registration block.
2. You can see the radio button with the option registration form and login form select any one of them and then select respective parameter's like you want login form or my-account form.
3.After that you can see the shortcode will be rendered as per configuration.
4. Test in Frontend whether it's working or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Gutenberg block for login.
